src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/InitialConsiderationService.java

```diff
@@ -105,20 +105,27 @@ public String getHearingDetails(List<HearingTypeItem> hearingCollection) {
             .filter(hearingTypeItem -> hearingTypeItem != null && hearingTypeItem.getValue() != null)
             .map(HearingTypeItem::getValue)
             .filter(
-                hearing -> hearing.getHearingDateCollection() != null && !hearing.getHearingDateCollection().isEmpty())
-            .min(
-                Comparator.comparing(
+                hearing -> hearing.getHearingDateCollection() != null
+                    && !hearing.getHearingDateCollection().isEmpty())
+            .min(Comparator.comparing(
                     (HearingType hearing) ->
                         getEarliestHearingDateForListedHearings(hearing.getHearingDateCollection()).orElse(
                             LocalDate.now().plusYears(100))))
-            .map(hearing -> String.format(HEARING_DETAILS,
-                getEarliestHearingDateForListedHearings(hearing.getHearingDateCollection())
-                    .map(formatter::format).orElse(""),
-                hearing.getHearingType(),
-                getHearingDuration(hearing)))
+            .map(hearing -> getFormattedHearingDetails(hearing, formatter))
             .orElse(HEARING_MISSING);
     }
 
+    private String getFormattedHearingDetails(HearingType hearing, DateTimeFormatter formatter) {
+        Optional<LocalDate> earliestHearingDate = getEarliestHearingDateForListedHearings(
+            hearing.getHearingDateCollection());
+        if (earliestHearingDate.isPresent()) {
+            return String.format(HEARING_DETAILS, earliestHearingDate.map(formatter::format).orElse(""),
```

Comment:
think the orElse is unnecessary.

Reply:
Thanks. It is to resolve the return type being optional.